stack.yaml.lock

```diff
@@ -3,31 +3,5 @@
 # For more information, please see the documentation at:
 #   https://docs.haskellstack.org/en/stable/lock_files
 
```

Comment:
This file can be removed entirely, stack will know not to recreate it on next build :+1: